src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,11 @@
+# DATABASE CONFIGURATION
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: ${POSTGRES_DB}
+    username: ${POSTGRES_USER}
+    password: ${POSTGRES_PASSWORD}
```

Comment:
this can be generalized and placed into `application.yml` file.
as 
```yml
  datasource:
    driver-class-name: org.postgresql.Driver
    url: ${SPRING_DATASOURCE_URL}
    username: ${SPRING_DATASOURCE_USERNAME}
    password: ${SPRING_DATASOURCE_PASSWORD}
```

then add .env.dev file 
and add this into application-dev.yml
```yaml
spring:
  config:
    import: optional:file:.env.dev[.properties]
```